src/libraries/System.Net.HttpListener/src/System/Net/WebSockets/HttpListenerWebSocketContext.cs

```diff
@@ -102,10 +102,8 @@ public class HttpListenerWebSocketContext : WebSocketContext
                 {
                     // AuthenticationSchemes.Digest, AuthenticationSchemes.Negotiate, AuthenticationSchemes.NTLM.
 #if (WINDOWS)
```

Comment:
```suggestion
#if TARGET_WINDOWS
```

it seems we use that way in other places. (may impact cross-targeting and cross-compilation)

Reply:
I tried it, but it seems unknown, here is a list of available windows preprocessor directives :
* WINDOWS
* WINDOWS7_0
* WINDOWS7_0_OR_GREATER

Reply:
It seems to be set explicitly in project files that need it.
https://github.com/dotnet/runtime/blob/959a13fa0d1822dc4c7bfc99a9d5e7a0c50d6306/src/libraries/System.Net.Security/src/System.Net.Security.csproj#L13